src/lib/assets/icons/reply.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If an icon is available in the Material UI icon set, we should use it instead.  I think this is ReplyTwoTone?  Today we access these via the IconSvg component.